index.html

```diff
@@ -10293,12 +10293,12 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
 			<pdef>aria-activedescendant</pdef>
 			<div class="property-description">
 				<p>Identifies the currently active element when DOM focus is on a <rref>composite</rref> widget, <rref>combobox</rref>, <rref>textbox</rref>, <rref>group</rref>, or <rref>application</rref>.</p>
-				<p>The <code>aria-activedescendant</code> property provides an alternative method of managing focus for interactive elements that may contain multiple focusable descendants, such as menus, grids, and toolbars. Instead of moving DOM focus among descendant elements, authors MAY set DOM focus on an <a>element</a> that supports <code>aria-activedescendant</code> and then use <code>aria-activedescendant</code> to refer to the element that is active.</p>
+				<p>The <code>aria-activedescendant</code> property provides an alternative method of managing focus for interactive elements that may contain multiple focusable descendants, such as menus, grids, and toolbars. Instead of moving DOM focus among <a>owned</a> elements, authors MAY set DOM focus on a container <a>element</a> that supports <code>aria-activedescendant</code> and then use <code>aria-activedescendant</code> to refer to the element that is active.</p>
 				<p> Authors MUST ensure that one of the following two sets of conditions is met when setting the value of <code>aria-activedescendant</code> on an element with DOM focus:</p>
 				<ol>
-					<li>The value of <code>aria-activedescendant</code> refers to an element that is either a descendant of the element with DOM focus or is a logical descendant as indicated by the <pref>aria-owns</pref> attribute.</li>
+					<li>The value of <code>aria-activedescendant</code> refers to an <a>owned</a> element, an owned element is either a descendant of the element with DOM focus or is a logical descendant as indicated by the <pref>aria-owns</pref> attribute.</li>
```

Comment:
Break into 2 sentences for clarity, and don't need the 2nd "is".

```suggestion
					<li>The value of <code>aria-activedescendant</code> refers to an <a>owned</a> element. An owned element is either a descendant of the element with DOM focus or a logical descendant as indicated by the <pref>aria-owns</pref> attribute.</li>
```